### PR TITLE
Release Flux v1.10.0

### PR DESCRIPTION
**Pull request by hiddeco**

### CODE REVIEW

**Comment by hiddeco** [comment]
Force-pushed to change the release date to tomorrow (2019-02-07).

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,59 @@
 This is the changelog for the Flux daemon; the changelog for the Helm
 operator is in [./CHANGELOG-helmop.md](./CHANGELOG-helmop.md).
 
+## 1.10.0 (2019-02-07)
+
+This release adds a `--registry-exclude-image` flag to the daemon to
```
Comment (squaremo) [inline CHANGELOG.md]: Reads a bit like the flag does all these things. I reckon:

This release adds the `--registry-exclude-image` flag for excluding images from scanning, allows for registries with self-signed certificates, and fixes several bugs.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,59 @@
...
+  [weaveworks/flux#1658][#1658]
+- Improved handling of registry challenge requests (preventing memory
+  leaks) [weaveworks/flux#1672][#1672]
+- Altered merging strategy for image credentials, which previously
+  could lead to Flux trying to fetch image details with credentials
+  from a different workload [weaveworks/flux#1702][#1702]
+
+### Improvements
+
+- Allow (all) images to be excluded from scanning
```
Comment (squaremo) [inline CHANGELOG.md]: "(potentially all)"

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,59 @@
...
+- Removed last traces of `linting` [weaveworks/flux#1673][#1673]
+- Warn users about external changes in sync tag
+  [weaveworks/flux#1695][#1695]
+
+### Thanks
+
+Lots of thanks to @2opremio, @alanjcastonguay, @bheesham, @brantb,
+@dananichev, @dholbach, @dmarkey, @hiddeco, @ncabatoff, @rade,
+@squaremo, @switchboardOp, @stefanprodan and @Timer for their
+contributions to this release, including those who I may have missed
```
Comment (squaremo) [inline CHANGELOG.md]: "including those" suggests you're clarifying a general statement -- for example "thanks to everyone who contributed, including those who logged issues", so it reads as slightly odd when you've already been specific.

Perhaps use "and anyone I've missed while writing this note."

**Pull request merged**